stable/ark/Chart.yaml

```diff
@@ -1,15 +1,14 @@
 apiVersion: v1
 appVersion: 0.10.2
-description: A Helm chart for ark
+## This Ark chart is deprecated because Ark has been renamed to Velero as of Velero version 0.11.0.
+## This chart is no longer maintained. Use the stable/velero chart for future work.
+## Please see the Helm deprecation policy in the PROCESSES.md file.
+deprecated: true
+description: DEPRECATED A Helm chart for ark
 name: ark
-version: 4.2.1
+version: 5.0.0
```

Comment:
We should just bump the patch version here.

Reply:
Its a completely new chart, shouldn't we start again at 1.0.0 for velero and leave ark at 4.2.1?

Reply:
Done